### PR TITLE
Support 'submit' input type in the crawler

### LINKED ISSUE #65: Support 'submit' input type in the crawler
Around line 159
`elif input_type == 'radio':
elem.click()`

### PR DESCRIPTION
**Pull request by yash-iiith**
Added an elif statement for 'submit' input type in collect.py

Fixes #65.

### CODE REVIEW

**Comment by yash-iiith** [comment]
@marco-c

File: collect.py [hunk 1]
```diff
@@ -158,6 +158,8 @@ def do_something(driver, elem_attributes=None):
             elem.send_keys('3')
         elif input_type == 'radio':
             elem.click()
+        elif input.type == 'submit':
```
Comment (marco-c) [inline collect.py]: This should be `input_type`.

**Comment by marco-c** [comment]
Could you upload here some example screenshots?

**Comment by marco-c** [comment]
> when I run get_dependencies.py, I encounter a syntax error:

Are you running it with Python 3? Python 3 is required for this project.

**Comment by yash-iiith** [comment]
Yeah. It ran successfully with Python 3. But running ```python3 collect.py``` gives me an error:
```ImportError: /usr/local/lib/python2.7/dist-packages/PIL/_imaging.so: undefined symbol: PyString_FromStringAndSize``` I tried changing the ```PYTHONPATH``` to ```/usr/local/lib/python3.5/dist-packages/``` but then it gives me ```ImportError: No module named 'selenium'``` even though selenium is installed in the system. Please guide me through this.

**Comment by yash-iiith** [comment]
I am now installing each of the packages for python3 as I have them installed for python2.7. Hoping that will solve the problem.

**Comment by yash-iiith** [comment]
897_firefox
897_6_firefox
897_5_firefox
897_4_firefox
897_3_firefox
897_2_firefox
897_1_firefox
897_0_firefox

**Comment by marco-c** [comment]
Can you upload screenshots for a page where there is a submit input? Upload the screenshot before pressing on the submit input and after.
If you can't find such a page, you can write a simple example yourself and make the crawler run on that example.

**Comment by marco-c** [comment]
Closing for lack of response. Please reopen if you still intend on working on this.

**Pull request closed without merging**